sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/StatsbeatConstants.cs

```diff
@@ -15,6 +15,7 @@ internal static class StatsbeatConstants
         {
             "francecentral",
             "francesouth",
+            "germanywestcentral",
```

Comment:
What's the expected behavior when the customer is using an older version of the SDK which doesn't have this line? (will the data be simply dropped?)

Reply:
Yes, We will not initialize statsbeat in such cases. The region has to exactly match either one of EU regions or NonEU regions.